lm75bd/lm75bd.c

```diff
@@ -27,7 +29,38 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
-  
+  if (temp == 0){
+    return ERR_CODE_INVALID_ARG; //pointer should not be null
+  }
+
+  uint8_t tempPointer = 0x0; //temp pointer reg is set to 0 to specify temperature reading to the device
+
+  error_code_t res = i2cSendTo(devAddr, &tempPointer, 1); // write pointer byte 0x0 to the device address for temperature reading
+
+  if (res != ERR_CODE_SUCCESS){
+    return ERR_CODE_UNKNOWN;
+  }
+
+  uint8_t buff[2] = {}; //0 is MSB, 1 is LSB
```

Comment:
Initialize the buffer by putting a 0 inside the curly brackets. This apply to other places were you just have curly brackets for structures but no 0 inside